slowquant/unitary_coupled_cluster/sa_ups_wavefunction.py

```diff
@@ -250,10 +250,10 @@ def __init__(
         elif ansatz.lower() == "qnp":
             self.ansatz_options["do_qnp"] = True
             self.ups_layout.create_tups(self.num_active_orbs, self.ansatz_options)
-        elif ansatz.lower() == "fucc":
-            self.ups_layout.create_fUCCSD(states[1], self.ansatz_options)
         elif ansatz.lower() == "ksafupccgsd":
```

Comment:
Why is fucc deleted`?

Reply:
It got deemed unworthy for SA, since it is not S2 conserving.
Together with having it for SA makes the implementation of it a lot more complicated.